### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fthobe**
Added some limitations to underline what’s possible and what not.

## What?
It is not clear in the documentation that google enhanced e-commerce analytics can not be used with hosted payment solutions. 

## Why?
Well, it would be nice if people know before starting implementation.

## Testing / Proof
Well, we just burned a week of development and mails with BC to figure it out, but give it a try: 
As external payment providers redirect the customer to payments. Bigcommerce.com and payments.bigcommerce.com redirects directly to the order confirmation page there’s no possibility to finalize the order on the checkout and you people should really tell us before! 

@bigcommerce/checkout @bigcommerce/payments

### CODE REVIEW

**Comment by fthobe** [comment]
@davidchin here you go with your documentation update.

**Comment by fthobe** [comment]
@travis-ci And for the first time in my life I made a low risk readme commit and my pull request fails :D

**Comment by icatalina** [comment]
Hi @fthobe, first of all, thanks for your contribution.

Travis is failing because your commit doesn't follow the commit message guidelines. We use the commit messages to generate our changelog, hence the strictness. It is all described in the `CONTRIBUTING.md` file: https://github.com/bigcommerce/checkout-sdk-js/blob/master/CONTRIBUTING.md#commit-guideline

File: README.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ The library also provides integrations with all the payment methods supported by
 
 Using this library in conjunction with your favorite UI framework, it is possible to build a bespoke checkout UI for a store, that can be augmented with additional features. We provide a basic [reference implementation](https://github.com/bigcommerce/checkout-sdk-js-example) of a custom checkout written in React to get you started.
 
+## Limitations
+* It is not possible to execute custom jscript when paying on hosted payment pages such as Adyen, therefore Google Enhenced eCommerce Analytics can not be fully used as payment confirmation is handled by payments.bigcommerce.com and not by the checkout.
```
Comment (icatalina) [inline README.md]:
```suggestion
* It is not possible to execute custom JavaScript when paying on hosted payment pages such as Adyen, therefore Google Enhanced eCommerce Analytics can not be fully used as payment confirmation is handled by payments.bigcommerce.com and not by the checkout.
```

Could you please provide some context on what you mean by _Google Enhanced eCommerce Analytics **can not be fully used**_? Thanks 🙇

Comment (fthobe) [reply README.md]: @icatalina Sorry, I just saw this comment. You are right, the more precise description would be

It is not possible to execute custom JavaScript when paying on hosted payment pages such as Adyen on the checkout after the payment has been completed, therefore conversion tracking solutions or any other task that needs to be completed after the payment need to implemented on the order confirmation page.

**Comment by animesh1987** [comment]
@fthobe is this still valid? I am going to close this PR for now

**Pull request closed without merging**